src/platform/unix.rs

```diff
@@ -299,10 +299,11 @@ impl WindowExtUnix for Window {
     #[inline]
     #[cfg(feature = "wayland")]
     fn wayland_set_csd_theme(&self, theme: Theme) {
+        #[allow(clippy::single_match)]
```

Comment:
Don't the other functions above need this too, which have a seemingly identical `match` block?

Reply:
No, because they return, but this does not.